packages/compass-app-stores/src/modules/instance/index.ts

```diff
@@ -0,0 +1,45 @@
+import { combineReducers } from 'redux';
+import type { ErrorMessageAction, ErrorMessageState } from './error-message';
+import errorMessage, {
+  INITIAL_STATE as ERROR_MESSAGE_INITIAL_STATE,
+} from './error-message';
+import type { InstanceAction, InstanceState } from './instance';
+import instance, { INITIAL_STATE as INSTANCE_INITIAL_STATE } from './instance';
+import type { DataServiceAction, DataServiceState } from './data-service';
+import dataService, {
+  INITIAL_STATE as DATA_SERVICE_INITIAL_STATE,
+} from './data-service';
+
+import type { ResetAction } from './reset';
+import { RESET } from './reset';
+
+export type RootState = {
+  dataService: DataServiceState;
+  errorMessage: ErrorMessageState;
+  instance: InstanceState;
+};
+export type RootAction =
+  | DataServiceAction
+  | ErrorMessageAction
+  | InstanceAction
+  | ResetAction;
+
+const reducer = combineReducers<RootState, RootAction>({
+  errorMessage,
+  instance,
+  dataService,
+});
+
+const rootReducer = (state: RootState | undefined, action: RootAction) => {
```

Comment:
Okay, maybe it's a bit too big of a change, but now I'm thinking we probably can just drop redux completely here? The instance can be created when activate is called and returned as `store: { instance }`, this should be enough to access it in the render method and pass to the provider in the component

Reply:
Yup, fully removed redux here now! :slightly_smiling_face: